### PR TITLE
Adding log level to command

### PR DESCRIPTION
**Pull request by vdesoutter**

### CODE REVIEW

File: test/integration.sh [hunk 1]
```diff
@@ -122,6 +122,23 @@ else
   exit 1
 fi
 
+rm -rf $CL_HOME/dropins
+mkdir -p $CL_HOME/dropins
+cp -R $SCRIPT_DIR/packages-src/bonjour $CL_HOME/dropins
+
+echo "> test the LOG_LEVEL command"
```
Comment (bhou) [inline test/integration.sh:129]: There are two tests with the same name: test the the LOG_LEVEL command

**Review by bhou — CHANGES_REQUESTED** [review]
Two tests with the same name

File: test/integration.sh (hunk 1 above)
Comment (gmonceyron) [reply test/integration.sh:129]:
```suggestion
echo "> test the LOG_LEVEL command - no log level from configuration"
```

File: test/integration.sh [hunk 2]
```diff
@@ -137,6 +154,16 @@ else
   exit 1
 fi
 
+echo "> test the LOG_LEVEL command"
```
Comment (gmonceyron) [inline test/integration.sh:157]:
```suggestion
echo "> test the LOG_LEVEL command - actual log level from configuration"
```

**Review by gmonceyron — CHANGES_REQUESTED** [review]

**Review by gmonceyron — APPROVED** [review]

**Pull request merged**